package.json

```diff
@@ -17,6 +17,7 @@
     "babel-runtime": "^6.11.6",
     "classnames": "^2.2.5",
     "crypto": "0.0.3",
```

Comment:
Maybe this line can be deleted?

Reply:
good call! removed in d40d2717670d71a5a45a8827efe86acfab9760a6